imports/plugins/included/product-variant/containers/productsContainerAdmin.js

```diff
@@ -159,7 +157,12 @@ function composer(props, onData) {
   }
 
   const queryParams = Object.assign({}, tags, Reaction.Router.current().query, shopIds);
-  const productsSubscription = Meteor.subscribe("Products", scrollLimit, queryParams, sort, editMode);
+  let productsSubscription = Meteor.subscribe("Products", scrollLimit, queryParams, sort, editMode);
+  if (resubscribeAfterCloning.get()) {
+    resubscribeAfterCloning.set(false);
+    productsSubscription.stop();
+    productsSubscription = Meteor.subscribe("Products", scrollLimit, queryParams, sort, editMode);
+  }
```

Comment:
Would this be enough?

```js
const productsSubscription = Meteor.subscribe("Products", scrollLimit, queryParams, sort, editMode);
if (resubscribeAfterCloning.get()) {
  resubscribeAfterCloning.set(false);
}
```

Doing the `Meteor.subscribe` call twice seems unnecessary. The important thing is triggering this whole function again through the reactive var. If you need to explicitly stop it first (before the first call) you could, but I think it stops itself due to being tracked anyway.